src/main/java/org/hl7/davinci/priorauth/FhirUtils.java

```diff
@@ -40,6 +40,9 @@
   public static final String WEBSOCKET_EXTENSION_URL = "http://hl7.org/fhir/StructureDefinition/capabilitystatement-websocket";
   public static final String SECURITY_SYSTEM_URL = "http://terminology.hl7.org/CodeSystem/v3-ObservationValue";
   public static final String SECURITY_SUBSETTED = "SUBSETTED";
+  public static final String ITEM_TRACE_NUM_STRING = "http://hl7.org/fhir/us/davinci-pas/StructureDefinition/extension-itemTraceNumber"; /*
+                                                                                                                                          * itemTraceNumber
+                                                                                                                                          */
```

Comment:
Just for consistency rename it. Comment isn't needed either then. You can also move this up to be in alphabetical order with the other extension URLs
```suggestion
  public static final String ITEM_TRACE_NUMBER_EXTENSION_URL = "http://hl7.org/fhir/us/davinci-pas/StructureDefinition/extension-itemTraceNumber"; 
```